data/src/main/java/me/androidbox/data/di/NetworkModule.kt

```diff
@@ -0,0 +1,68 @@
+package me.androidbox.data.di
+
+import dagger.Module
+import dagger.Provides
+import dagger.Reusable
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import me.androidbox.data.BuildConfig
+import me.androidbox.data.remote.interceptor.ApiKeyInterceptor
+import me.androidbox.data.remote.network.authentication.AuthenticationService
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+@Module
+@InstallIn(SingletonComponent::class)
```

Comment:
Is this correct that it should be singleton?

Reply:
Yes!